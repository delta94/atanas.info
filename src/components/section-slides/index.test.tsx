/* eslint-disable @typescript-eslint/ban-ts-comment */

import * as React from 'react';
import { act } from 'react-dom/test-utils';
import { mount } from 'enzyme';

import { SectionSlides } from '.';

describe('SectionSlides component', () => {
	it('Should render the SectionSlides component', () => {
		act(() => {
			const wrapper = mount(<SectionSlides />);

			expect(wrapper).toMatchSnapshot();

			// @ts-ignore
			wrapper.find('button').at(0).props().onClick();

			expect(wrapper).toMatchSnapshot();
		});
	});
});
