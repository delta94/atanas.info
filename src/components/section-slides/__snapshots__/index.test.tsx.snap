// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionSlides component Should render the SectionSlides component 1`] = `
<Component
  hasButton={true}
  hasShell={true}
  id="slides"
>
  <h2>
    Slides
  </h2>
  <nav
    className="c-section__nav"
  >
    <ul>
      <li
        className="current"
        key="0"
      >
        <Component
          onClick={[Function]}
          type="button"
        >
          EcmaScript Features
        </Component>
      </li>
      <li
        className=""
        key="1"
      >
        <Component
          onClick={[Function]}
          type="button"
        >
          Do more with Sublime Text
        </Component>
      </li>
    </ul>
  </nav>
  <div
    className="c-section__body"
  >
    <div
      className="c-section__frame current"
      key="0"
    >
      <iframe
        allowFullScreen={true}
        src="https://docs.google.com/presentation/d/e/2PACX-1vSA8qN0_3lb0YWDYJheJ3q0AAsXWlLLpP86pnmV0KWfPo_2G-SSdSK--GVRyJbwT8_kWyJ0TZxFoaH8/embed?start=false&loop=false&delayms=3000"
        title="Presentation about the EcmaScript language history and features"
      />
    </div>
    <div
      className="c-section__frame"
      key="1"
    >
      <iframe
        allowFullScreen={true}
        src="https://docs.google.com/presentation/d/e/2PACX-1vTivcVmPyjYPdWgoJMChe_rrVyZEIw4ZgF37lCOlxUUm7DchxhMJouEV_BcrH6LDL8kiNmRSIWkiSxK/embed?start=false&loop=false&delayms=3000"
        title="Presentation about improving workflow using Sublime Text editor"
      />
    </div>
  </div>
</Component>
`;
